ui/lib/core/addon/components/enable-input.hbs

```diff
@@ -11,7 +11,7 @@
       {{#if @attr}}
         <ReadonlyFormField @attr={{@attr}} @value="**********" />
       {{else}}
-        <Input disabled class="input" @type="text" @value="**********" />
+        <Input aria-label={{@label}} readonly class="input" @type="text" @value="**********" />
```

Comment:
Updating this to readonly instead of disabled might be going too far, since we used disabled many other places. Let me know if it feels like too much and I'll just add the label stuff

Reply:
I thought `aria-label` was sufficient to satisfy accessibility? But I suppose I don't fully understand why readonly might be easier to apply than disabled